engine/apps/chatops_proxy/tasks.py

```diff
@@ -120,3 +123,47 @@ def unlink_slack_team_async(**kwargs):
             f'msg="Failed to unlink slack_team: {e}" service_tenant_id={service_tenant_id} slack_team_id={slack_team_id}'
         )
         raise e
+
+
+@shared_dedicated_queue_retry_task(
+    autoretry_for=(Exception,),
+    retry_backoff=True,
+    max_retries=0,
+)
+def start_sync_org_with_chatops_proxy():
+    from apps.user_management.models import Organization
+
+    organization_qs = Organization.objects.all()
+    organization_pks = organization_qs.values_list("pk", flat=True)
+
+    max_countdown = 60 * 30  # 30 minutes, feel free to adjust
+    for idx, organization_pk in enumerate(organization_pks):
+        countdown = idx % max_countdown
+        sync_org_with_chatops_proxy.apply_async((organization_pk,), countdown=countdown)
+
+
+@shared_dedicated_queue_retry_task(
+    autoretry_for=(Exception,),
+    retry_backoff=True,
+    max_retries=3,
+)
+def sync_org_with_chatops_proxy(**kwargs):
+    from apps.user_management.models import Organization
+
+    organization_id = kwargs.get("organization_id")
+    task_logger.info(f"sync_org_with_chatops_proxy: started org_id={organization_id}")
+
+    try:
+        org = Organization.objects.get(pk=organization_id)
+    except Organization.DoesNotExist:
+        task_logger.info(f"Organization {organization_id} was not found")
+        return
+
+    try:
+        register_oncall_tenant(org)
+    except ChatopsProxyAPIException as api_exc:
+        # TODO: once tenants upsert api is released, remove this check
```

Comment:
saw a similar comment [here](https://github.com/grafana/oncall/pull/4559/files#diff-da2e03e43e5bbdf0accf835301a5a6c9e84e8734862c215a9ad23d8d4ae3fba4R30) in #4559. What is the upsert API?